lib/bemxjst/index.js

```diff
@@ -426,13 +426,13 @@ BEMXJST.prototype.tryRun = function(context, ent) {
   try {
     return ent.run(context);
   } catch (e) {
-    console.error('BEMXJST ERROR: cannot render ' +
-      [
-        'block ' + context.block,
-        'elem ' + context.elem,
-        'mods ' + JSON.stringify(context.mods),
-        'elemMods ' + JSON.stringify(context.elemMods)
-      ].join(', '), e);
+    var error = this.context.onError || console.error;
+    error({
+      block: context.block,
+      elem: context.elem,
+      mods: context.mods,
+      elemMods: context.elemMods
```

Comment:
1. @sbmaxx asked me. I aggree.
2. Better to have normalized information. In `context.ctx` may be not `block` field or etc.